tools/docker/docker-compose-mac.yml

```diff
@@ -107,6 +107,7 @@ services:
       retries: 10
     volumes:
       - ${EDA_HOST_PODMAN_SOCKET_URL:-/run/user/501/podman/podman.sock}:/run/podman/podman.sock:z
+      - '../../:/app/src:z'
```

Comment:
@Dostonbek1 What would the ../.. be relative to if we are in the root directory of the project? Should we have a SRC_DIR var defined in the docker-compose file

Reply:
@mkanoor it is relative to the location of the compose file. It is also in line with the rest of the compose files. No need any variable.